css/types/calc.json

```diff
@@ -301,21 +301,19 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": "57",
-                "version_removed": "59",
-                "notes": "<code>calc()</code> was temporarily disabled in media query values due to the changeover to Stylo. In Firefox 59, support was added back in again."
+                "version_added": "59"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "59"
```

Comment:
Please keep it false for the moment. Stylo is not yet in Firefox for Android, and it is unclear if it will be in 59 or 60.